src/select/select.js

```diff
@@ -312,13 +331,18 @@ class Select extends React.Component<PropsT, StatelessStateT> {
                   });
                   e.stopPropagation();
                 }}
+                {...tagProps}
               >
                 {this.getSelectedOptionLabel(option)}
               </Tag>
             ) : (
-              <StyledSingleSelection key={option.id}>
+              <SingleSelection
+                key={option.id}
+                $disabled={disabled}
```

Comment:
seems like the `$disabled` prop should be passed to all underlying styled components